cpp/cmake/Dependencies.cmake

```diff
@@ -126,17 +126,14 @@ set(TREELITE_DIR ${CMAKE_CURRENT_BINARY_DIR}/treelite CACHE STRING
   "Path to treelite install directory")
 ExternalProject_Add(treelite
     GIT_REPOSITORY    https://github.com/dmlc/treelite.git
-    GIT_TAG           6fd01e4f1890950bbcf9b124da24e886751bffe6
+    GIT_TAG           e776e202fa732816e0a617715250fc727d573edf
     PREFIX            ${TREELITE_DIR}
-    CMAKE_ARGS        -DBUILD_SHARED_LIBS=OFF
-                      -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
-                      -DENABLE_PROTOBUF=ON
+    CMAKE_ARGS        -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
                       -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}
-    BUILD_BYPRODUCTS  ${TREELITE_DIR}/lib/libtreelite.a
+    BUILD_BYPRODUCTS  ${TREELITE_DIR}/src/treelite-build/libtreelite_static.a
                       ${TREELITE_DIR}/lib/libdmlc.a
-                      ${TREELITE_DIR}/lib/libtreelite_runtime.so
-    UPDATE_COMMAND    ""
-    PATCH_COMMAND     patch -p1 -N < ${CMAKE_CURRENT_SOURCE_DIR}/cmake/treelite_protobuf.patch || true)
+                      ${TREELITE_DIR}/src/treelite-build/libtreelite_runtime.so
+    UPDATE_COMMAND    "")
```

Comment:
You got rid of the PATCH_COMMAND! Awesome.

Reply:
I also got rid of the protobuf dependency.